SmartDeviceLink-iOS/SmartDeviceLink/SDLTCPTransport.m

```diff
@@ -63,8 +63,10 @@ - (void)connect {
 
 - (void)sendData:(NSData *)msgBytes {
     dispatch_async(_sendQueue, ^{
-        NSString *byteStr = [SDLHexUtility getHexString:msgBytes];
-        [SDLDebugTool logInfo:[NSString stringWithFormat:@"Sent %lu bytes: %@", (unsigned long)msgBytes.length, byteStr] withType:SDLDebugType_Transport_TCP toOutput:SDLDebugOutput_DeviceConsole];
+        @autoreleasepool {
+            NSString *byteStr = [SDLHexUtility getHexString:msgBytes];
+            [SDLDebugTool logInfo:[NSString stringWithFormat:@"Sent %lu bytes: %@", (unsigned long)msgBytes.length, byteStr] withType:SDLDebugType_Transport_TCP toOutput:SDLDebugOutput_DeviceConsole];
+        }
```

Comment:
What is the reasoning behind this? Not saying it's incorrect, just curious about your logic.